docs/fluvio/cli/fluvio/hub/download.mdx

```diff
@@ -14,15 +18,11 @@ After downloading, all SmartModules, regardless of their source, are treated the
 
 Downloading a SmartModule From the Hub is as follows:
 
-```bash copy="fl"
-$ fluvio hub download infinyon/json-sql@0.1.0
+<CodeBlock language="bash" copyBehavior="fl">$ {SmDownloadJsonSql}</CodeBlock>
 
-trying connection to fluvio router.infinyon.cloud:9003
-downloading infinyon/json-sql@0.1.0 to infinyon-json-sql-0.1.0.ipkg
-... downloading complete
-... checking package
-... cluster smartmodule install complete
-```
+Expect the following output:
+
+<CodeBlock language="bash">$ {SmDownloadJsonSqlOutput}</CodeBlock>
```

Comment:
I think that we don't need the "$" for output

Reply:
Gotcha! Thanks @fraidev!